service/history/outbound_queue_standby_task_executor.go

```diff
@@ -126,8 +136,11 @@ func (e *outboundQueueStandbyTaskExecutor) Execute(
 			task,
 			e.Now,
 			e.config.StandbyTaskMissingEventsResendDelay(task.GetType()),
```

Comment:
Please remove this from here for now since it doesn't do anything and is misleading.